.github/workflows/updatejson.yml

```diff
@@ -0,0 +1,27 @@
+name: Housekeeping
+on: 
+  push:
+  schedule:
+  #  Kick off job at midnight on the 1st and the 15th of every month
+  - cron: "0 0 1,15 * *"
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps: 
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - id: run_py
+        run:  |
+        cd utils
+        python update_json_files
```

Comment:
@PhilipWright96 Actually you also want to copy the books.json file to `app/src/data/books.json` after the json files are updated. You can also possibly update the python script to save the file there instead as well instead of adding a copy command here.

Reply:
Done - I just added a cp command in the workflow itself - that seemed easiest to me. Let me know if thats alright or not